src/release/process.md

```diff
@@ -129,11 +134,15 @@ Decide on a time to do the release, T.
   After this [Update thanks.rust-lang.org][update-thanks] by triggering a build
   on GitHub Actions on the master branch.
 
-- **T-5m** - Merge blog post.
+- **T-2m** - Merge blog post.
 
 - **T** - Tweet and post everything!
 
-  - Twitter [@rustlang](https://twitter.com/rustlang)
+  - Twitter [@rustlang](https://twitter.com/rustlang). Optional template:
+
+    > Rust `$VERSION` has been released, featuring `$MAJOR_FEATURES`! 🎉 Check
+    > out the highlights on our blog: `$LINK`
+
   - Reddit [/r/rust](https://www.reddit.com/r/rust/)
   - [Hacker News](https://news.ycombinator.com/)
   - [Users forum](https://users.rust-lang.org/)
```

Comment:
I think we should drop everything but the users forum here as well while we're at it; in practice I don't think that the Rust team should be posting to these other forums as part of the release (though of course there's no ban on doing so).

Reply:
Removed.